chatcommands.py

```diff
@@ -1843,7 +1843,8 @@ def report_posts(urls, reported_by, reported_in=None, blacklist_by=None, operati
         if scan_spam and operation != "report-direct":
             comment = report_info + scan_why.lstrip()
             handle_spam(post=post, reasons=scan_reasons, why=comment)
-            Tasks.later(Metasmoke.post_auto_comment, comment, reported_by, url=url, after=300)
+            if custom_reason:
```

Comment:
I think we can probably drop the `after=300` down to some small value here - say 10.

Reply:
I am also confused why the original implementation would like to wait for 5 minutes... I am not even sure how the original implementation managed to work as intended. It should not, as it passes the comment data instead of the custom_reason data to the post_auto_comment() call, hence should post a comment like "manually reported by *user* in room *1234* for reasons *reason*" instead of "reason".

Reply:
However I do believe that there are some good reasons to wait that long when people first implemented it.

Reply:
I believe the threshold should be reduced to 30 or lower, per ArtOfCode.